web/styles/utils.sass

```diff
@@ -10,7 +10,7 @@
     height: 100%
 
 .container-width
-  width: 90%
+  width: 100%
```

Comment:
doesn't this default to 100% if we remove `width`?

Reply:
no it defaults to the width of the contents inside the container. We want the containers to occupy 100% of the width even if the content don't necessarily fill up 100% of the width.

Without `width: 100%`:
![no-width](https://user-images.githubusercontent.com/21287547/45011720-d106bd80-afc8-11e8-8110-18e15d685cee.png)

With `width: 100%`:
![width-100](https://user-images.githubusercontent.com/21287547/45011727-d8c66200-afc8-11e8-9c55-88a456888b84.png)